sphinx/domains/python.py

```diff
@@ -679,7 +679,29 @@ def get_index_text(self, modname: str, name_cls: Tuple[str, str]) -> str:
             else:
                 return name
 
-        return _('%s (%s attribute)') % (attrname, clsname)
+        return self.fmt % (attrname, clsname)
+
+
+class PyProperty(PyAttribute):
```

Comment:
Please inherit `PyObject` instead. PyProperty is not a kind of PyAttribute. For example, it does not support `value` option.

Reply:
Okay, makes sense!